discord/enums.py

```diff
@@ -912,6 +913,12 @@ class SubscriptionStatus(Enum):
     inactive = 2
 
 
+class StatusDisplayType(Enum):
+    name = 0  # pyright: ignore[reportAssignmentType]
```

Comment:
Did we figure out how this looks from a user's POV when they try using this attribute?

Reply:
Still the same as [before](https://github.com/Rapptz/discord.py/pull/10227#issuecomment-3063512685) when accessing `.name` or `.value` from the member, and no response to [the comment](https://github.com/microsoft/pyright/issues/9728#issuecomment-3063808515) left on the Pyright repository about it.

But to be fair, it works fine when setting or comparing it:

```py
act = discord.Activity(
    ...
    status_display_type=discord.StatusDisplayType.name,
)
print(act.status_display_type) # StatusDisplayType.name
if act.status_display_type is discord.StatusDisplayType.name:
    print("Status display type is name")
```